packages/core/src/actions/sendTransaction.ts

```diff
@@ -78,37 +74,10 @@ export async function sendTransaction<
       connector,
     })
 
-  const { connector: activeConnector } = getAccount(config)
-
-  const gas = await (async () => {
-    // Skip gas estimation if `data` doesn't exist (not a contract interaction).
-    if (!('data' in parameters) || !parameters.data) return undefined
-
-    // Skip gas estimation if connector supports simulation.
-    if ((connector ?? activeConnector)?.supportsSimulation) return undefined
-
-    // Skip gas estimation if `null` is provided.
-    if (gas_ === null) return undefined
```

Comment:
@jxom @tmm This is breaking, `viem` throws error if gas is set to `null`:
```
Details: Cannot convert null to a BigInt
Version: viem@2.21.51
    at getTransactionError (getTransactionError.js:18:12)
    at sendTransaction (sendTransaction.js:219:105)
    at async sendTransaction (sendTransaction.js:24:18)Caused by: TypeError: Cannot convert null to a BigInt
    at BigInt (<anonymous>)
    at numberToHex (toHex.js:135:19)
    at formatTransactionRequest (transactionRequest.js:37:89)
    at sendTransaction (sendTransaction.js:114:29)
    at async sendTransaction (sendTransaction.js:24:18)
```
Documentation should also be updated:
<img width="768" alt="Screenshot 2024-11-27 at 10 00 28" src="https://github.com/user-attachments/assets/7025bc54-7b10-408b-818c-7b60170a4f30">